[2023-10-11 15:40:37,880] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:37:23.955441+00:00 [queued]>
[2023-10-11 15:40:37,921] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:37:23.955441+00:00 [queued]>
[2023-10-11 15:40:37,922] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 15:40:37,922] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2023-10-11 15:40:37,922] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 15:40:37,944] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): data_validation> on 2023-10-11T15:37:23.955441+00:00
[2023-10-11 15:40:37,959] {standard_task_runner.py:52} INFO - Started process 8490 to run task
[2023-10-11 15:40:37,966] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tf_pipeline', 'data_validation', '2023-10-11T15:37:23.955441+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpee2aghyz', '--error-file', '/tmp/tmp8q8lz576']
[2023-10-11 15:40:37,973] {standard_task_runner.py:77} INFO - Job 10: Subtask data_validation
[2023-10-11 15:40:38,060] {logging_mixin.py:104} INFO - Running <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:37:23.955441+00:00 [running]> on host 0e092fd43c1f
[2023-10-11 15:40:38,170] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tf_pipeline
AIRFLOW_CTX_TASK_ID=data_validation
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T15:37:23.955441+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-11T15:37:23.955441+00:00
[2023-10-11 15:40:38,177] {python.py:151} INFO - Done. Returned value was: None
[2023-10-11 15:40:38,202] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=tf_pipeline, task_id=data_validation, execution_date=20231011T153723, start_date=20231011T154037, end_date=20231011T154038
[2023-10-11 15:40:38,272] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-11 15:40:38,336] {local_task_job.py:151} INFO - Task exited with return code 0
